app/assets/stylesheets/sticky-nav.scss

```diff
@@ -154,14 +154,6 @@
     font-weight: bold;
     text-align: center;
     font-stretch: condensed;
-    box-shadow: $light-shadow;
     margin-bottom: 5px;
-    &:hover {
-      box-shadow: $dark-shadow;
-      padding: 7px 39px;
```

Comment:
Yes! Remove those awkward values ("7px 39px") 🙌🏼

Reply:
Haha, just a coincidence (for now) :D